packages/decentraland-ecs/src/decentraland/Input.ts

```diff
@@ -28,7 +29,7 @@ export type InputState = Record<
 export type LocalPointerEvent = GlobalInputEventResult & {
```

Comment:
With the action button refactor, this name becomes inconsistent. Suggestion, rename to `LocalActionButtonEvent`.
```suggestion
export type LocalActionButtonEvent = GlobalInputEventResult & {
```